### DISCUSSION TITLE
What would you like to see this group focus on?

### DISCUSSION THREAD

**Discussion by ghost** [finos/InnerSource#13] [Q&A]
Over the last few meetings we have had some great conversations around our patterns, blockers and successes with InnerSource within our organisations. Please use this discussion to post your ideas on what this group should focus on over the next quarter. Please provide your ideas below, and use GitHub reactions on the comments to vote.

Some ideas we have received already: 
- Creation of an InnerSource maturity model (#12)
- How to measure InnerSource success with concrete examples. Suggested [here](https://github.com/finos/InnerSource/issues/10#issuecomment-819592400). Including Metrics, value proposition.
- Integrating InnerSource with OpenSource. Suggested [here](https://github.com/finos/InnerSource/issues/10#issuecomment-819652403)
- Common policies / guidelines / principles around InnerSource
- Getting started - best projects
- Specific tips/guidelines e.g. how when is a PR too big?
- How can we best advocate for InnerSource?
- Tooling: What tools and settings are commonly used? / What supporting code/tools do we have that could be open sourced?/ Feedback for tools vendors

Please add your interest for these ideas below, or submit other ideas you may have!

**Comment by godds** [comment]
Infographic of InnerSource vs OpenSource (pros/cons/differences)

_Mentioned at https://github.com/finos/InnerSource/issues/18_

**Reply by mcleo-d** [reply]
https://github.com/finos/open-source-readiness/issues/3

**Comment by godds** [comment]
[Awesome-style list](https://github.com/sindresorhus/awesome) with links to relevant resources and tools

**Reply by amaltson** [reply]
Love it, I think this could really help people get started on their InnerSource journey. We can provide docs, books, presentations, and tools.

**Comment by mcleo-d** [comment]
Pioneering InnerSource Case Studies for the Financial Service Industry - https://github.com/finos/InnerSource/discussions/24

**Comment by claredillon** [comment]
Following up on a conversation in one of the working group meetings from the maturity matrix - it was suggested we should have explicit examples of some of the resources mentioned there e.g.:

- Example ("non-trivial") README
- Example Contribution Guidelines
- Example SLA for inquiries, responses and code reviews (including perhaps the 90 day warranty pattern)

**Comment by claredillon** [comment]
In one of the other SIG meetings - an additional suggestion was made - to focus on how organizations can effectively work on InnerSource projects with third party vendors and to share relevant resources e.g. procurement guidelines, sample contracts.
